Makd.mak

```diff
@@ -745,6 +745,11 @@ test: $(test)
 .PHONY: fasttest
 fasttest: $(fasttest)
 
+# Phony rule to build all packages
+.PHONY: pkg
+pkg += $(patsubst $(PKG)/%.pkg,$O/pkg-%.stamp,$(PKG_FILES))
+pkg: $(pkg)
+
```

Comment:
Just for clarity, I would put the variable definition before the `.PHONY: pkg`, this is the style used across the file (group the target definition with the declaration of being a phony target).